### PR TITLE
SFR-1694: Refactor click actions in Playwright tests

### PR DESCRIPTION
**Pull request by clarissarichard**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1694)

### This PR does the following:
- Created a single function that accepts a locator as a parameter and clicks on it
- Refactored Cucumber/Playwright tests so that all click actions/events use the same function

### Testing requirements & instructions: 
-  Run Playwright tests and all tests pass

### CODE REVIEW

File: playwright/support/mappings.ts [hunk 1]
```diff
@@ -18,3 +18,22 @@ export const pages: { [name: string]: Pages } = {
...
+  "first login for options button": "text=Log in for options >> nth=0",
+  "government documents checkbox":
+    "//span[contains(text(), 'Show only US government documents')]",
+  "Latin language checkbox": "//label[contains(.,'Latin')]",
+  "year filter button": "#year-filter-button",
+  "first read online button": "text=Read Online >> nth=0",
+  "first request button":
+    "[href='https://www.nypl.org/research/collections/shared-collection-catalog/hold/request/b10715506-i13895605']",
+  "English language checkbox":
+    "//*[@id='languages-checkbox-group-0-wrapper']/label/span[1]",
```
Comment (samanthaandrews) [inline playwright/support/mappings.ts:37]: Is there potentially a better/less potentially flaky locator we could access here? (this is arguably the hardest part of these Playwright tests, and i know in many instances, there just isn't something better!)

For this one in particular, I'm thinking something like `//input[text()='English']` might work. Lmk what you find.

**Review by samanthaandrews — APPROVED** [review]
I left one comment to see if we can try to get that locator replaced with something better, but overall this looks good and I think it's good to merge as is! Feel free to try to replace that locator with something else or not, up to you.

**Comment by samanthaandrews** [comment]
This is a really great refactor. So much cleanup!!

File: playwright/support/mappings.ts (hunk 1 above)
Comment (clarissarichard) [reply playwright/support/mappings.ts:37]: Yes! I've been looking into optimizing these as well. I had changed some elements but not all of them. I'll also update the "Latin language checkbox" when I find something that works.

Comment (clarissarichard) [reply playwright/support/mappings.ts:37]: @samanthaandrews I tried a few combinations (`input` looks to be more of a sibling to the text) and the two CSS locators that seem to work are `span:text('English')` and `[id='languages-checkbox-group-0-wrapper']`. Unless `span` and `id` are the flaky parts of the locator?

**Review by ayan1229 — APPROVED** [review]

**Pull request merged**